packages/arb-token-bridge-ui/src/util/EthDepositUtils.ts

```diff
@@ -55,7 +61,27 @@ export async function depositEthEstimateGas(
     return fetchFallbackGasEstimatesForOrbitChainWithCustomFeeToken()
   }
 
-  // todo: update this when we support custom destination addresses for eth deposits
+  if (destinationAddress) {
```

Comment:
This check should also consider that the `destinationAddress` provided in params is different from the `address`, only then enter this flow. 

Otherwise, if `destinationAddress` and `address` are same, shouldn't it be considered a normal ETH deposit?